.github/workflows/add-docs-preview-link.yml

```diff
@@ -0,0 +1,18 @@
+name: Docs Preview Link
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    paths: docs/**
+jobs:
+  docs-preview-link:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
```

Comment:
nit: is this permission required? this github action only acts on pull-requests

Reply:
```suggestion
```